package.json

```diff
@@ -117,7 +117,6 @@
   "dependencies": {
     "atom-linter": "^9.0.1",
     "atom-package-deps": "^4.0.1",
```

Comment:
Up the minimum version here to `^4.5.0` and change the `package-deps` for Linter to `"linter:2.0.0"`. This will ensure that users have at least version 2.0.0 of Linter installed.